korgan/response_legal.py

```
@@ -0,0 +1,320 @@
+            mentioned = _article_numbers_in_text(item)
+            if mentioned and not mentioned.issubset(allowed_articles):
+                notes.append("Исключено из правового обоснования как неподтверждённая точная норма: " + item)
+                continue
+            if any(marker in item.lower() for marker in _FORBIDDEN_BODY_MARKERS):
+                notes.append("Исключён служебный/ссылочный текст из правового обоснования.")
+                continue
+            legal_basis.append(item)
+
+        draft = ResponseToClaimDraft(
+            status=research.status,
+            title=str(payload.get("title", "ОТЗЫВ НА ИСК")).strip() or "ОТЗЫВ НА ИСК",
```

Ruff (0.16.1)

[warning] String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)